tools/ci/init

```diff
@@ -196,9 +195,9 @@ def move_java_files(output_path: str, framework_name: str):
 
 
 def update_files(output_path: str, framework_name: str):
-
-    template_pattern = "([^-]?)template([^_]?)"
-    framework_pattern = "\\1{}\\2".format(framework_name)
+    # Replace all instances of "template" but the ones that look like shell
+    # flags named "template", e.g. -template=false.
+    template_pattern = "(?<!-)template"
```

Comment:
There are also function names with `template` in them which we should not replace.
I would make the pattern: `"(?<!-|_)template(?!_)"`
The following should then not be matched:
* `-template`
* `function_template`
* `template_function`
* `another_template_function`

(we should write some unit tests for this!)